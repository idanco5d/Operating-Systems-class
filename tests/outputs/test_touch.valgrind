==30387== Memcheck, a memory error detector
==30387== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==30387== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==30387== Command: /mnt/c/Users/illus/os/hw/1/smash
==30387== Parent PID: 30382
==30387== 
==30387== 
==30387== FILE DESCRIPTORS: 4 open (3 std) at exit.
==30387== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_touch.valgrind
==30387==    <inherited from parent>
==30387== 
==30387== 
==30387== HEAP SUMMARY:
==30387==     in use at exit: 0 bytes in 0 blocks
==30387==   total heap usage: 115 allocs, 115 frees, 84,210 bytes allocated
==30387== 
==30387== All heap blocks were freed -- no leaks are possible
==30387== 
==30387== For lists of detected and suppressed errors, rerun with: -s
==30387== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
