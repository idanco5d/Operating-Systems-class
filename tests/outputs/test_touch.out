smash> smash> "2023-05-13 23:58:38.118625061 +0300,
smash> smash> "2023-05-13 23:58:38.138625062 +0300,
smash> 